tests/manager/deleteCustomer/managerCanDeleteCustomer.spec.js

```diff
@@ -1,27 +1,32 @@
 import { test } from '@playwright/test';
 import { faker } from '@faker-js/faker';
+import { AddCustomerPage } from '../../../src/pages/manager/AddCustomerPage';
+import { CustomersListPage } from '../../../src/pages/manager/CustomersListPage';
 
-test.beforeEach( async ({ page }) => {
-  /* 
-  Pre-conditons:
-  1. Open Add Customer page
-  2. Fill the First Name.  
-  3. Fill the Last Name.
-  4. Fill the Postal Code.
-  5. Click [Add Customer].
-  */
+let createdFirstName;
+let createdLastName;
 
+test.beforeEach(async ({ page }) => {
+  const addCustomerPage = new AddCustomerPage(page);
+  createdFirstName = faker.person.firstName();
+  createdLastName = faker.person.lastName();
+  const postCode = faker.location.zipCode();
+
+  await addCustomerPage.open();
+  await addCustomerPage.fillFirstNameField(createdFirstName);
+  await addCustomerPage.fillLastNameField(createdLastName);
+  await addCustomerPage.fillPostCodeField(postCode);
+  await addCustomerPage.clickAddCustomerButton();
```

Comment:
I would add some assertion after creating some test data. For example here you could check that customer is really created